cmd/access/access_node_builder.go

```diff
@@ -0,0 +1,200 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/rs/zerolog"
+
+	"github.com/onflow/flow-go/cmd"
+	"github.com/onflow/flow-go/cmd/build"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+	"github.com/onflow/flow-go/module/local"
+	"github.com/onflow/flow-go/network"
+	jsoncodec "github.com/onflow/flow-go/network/codec/json"
+	"github.com/onflow/flow-go/network/p2p"
+	"github.com/onflow/flow-go/network/topology"
+	"github.com/onflow/flow-go/network/validator"
+)
+
+// AccessNodeBuilder is initializes and runs an Access node either as a staked node or an unstaked node
+// It is composed of the FlowNodeBuilder
+type AccessNodeBuilder struct {
+	*cmd.FlowNodeBuilder
+	staked                  bool
```

Comment:
I wonder if there is a different term than "unstaked" that we can use for this new mode of operation for the Access Node. Being unstaked doesn't really describe the new mode of operation very well IMO, and the notions of stake and weight are already a bit overloaded and frankly confusing in parts in the protocol code (see [here](https://github.com/onflow/flow-go/blob/c59b5e4de1b59766d352209dbc856068c03f9225/model/flow/identity.go#L30-L39) for an example). 

For example, in the [`NodeConfig`](https://github.com/onflow/flow-go/pull/911/files#diff-7540be57ca1fb2714cfce630ad0eba7bfc97362d8e30b79ac2551dc68792e75bR252) we have a field called `Stake`, which really means `Weight` and we have a field called `Unstaked` which only applies to access nodes, doesn't have much to do with the field called `Stake`, and determines whether an AN runs in this new external network or as part of the core Flow network.

Some ideas: External AN / network, Proxy AN / network, Adjunct AN / network

Reply:
Public network? (versus our current private network)

Reply:
tbh - i like the word `unstaked network` it succinctly describes this network as a network of unstaked actors. Any other name seems to be requiring more explanation since our _flow network_ is public and external.

Reply:
Maybe in places where the new field / name is near an existing stake/weight reference we can expand the name to differentiate or add some comments to differentiate